pkg/adapter/apiserver/adapter.go

```diff
@@ -46,10 +46,11 @@ type apiServerAdapter struct {
 
 	config Config
 
-	discover discovery.DiscoveryInterface
-	k8s      dynamic.Interface
-	source   string // TODO: who dis?
-	name     string // TODO: who dis?
+	discover              discovery.DiscoveryInterface
+	k8s                   dynamic.Interface
+	source                string // TODO: who dis?
```

Comment:
Who dis?

Reply:
this is old... I think it was for metrics, not part of this change